### DISCUSSION TITLE
Unable to verify Oracle account...

### DISCUSSION THREAD

**Discussion by choimiyeo** [dbeaver/dbeaver#21719] [Q&A]
It has the following problems
![error1](https://github.com/dbeaver/dbeaver/assets/149549884/945658be-a0ee-4c62-8945-1f93eabfa310)

I upgraded my existing Devay to 23.2.3 and set it to Oracle Instantclient 21.3 and the schema changed like this.

I don't know what to do. Please help me!

**Comment by LonwoLonwo** [comment]
Hello @choimiyeo 

Probably, this is the driver issue. It doesn't return real schema names.
Could you please try this solution? https://github.com/dbeaver/dbeaver/issues/15144#issuecomment-1011636822
